app/javascript/controllers/select_date_controller.js

```diff
@@ -3,11 +3,22 @@
 // The verbose use of the `jQuery` variable instead of `$` is intentional so its use is easier to identify
 import { Controller } from 'stimulus'
 
+function init (element) {
+  jQuery(element).daterangepicker({
+    singleDatePicker: true,
+    showDropdowns: true,
+    autoUpdateInput: false
+  })
+}
+
+document.addEventListener('cable-ready:after-morph', () => {
+  document
+    .querySelectorAll('[data-controller="select-date"]')
+    .forEach(el => init(el))
+})
+
```

Comment:
are we leveraging any of the changes in this file?

Reply:
No but it's a good refactor. I say we leave it.